### ISSUE TITLE
SimpleReport - Validation Errors COVID Reporting - Missouri

### ISSUE THREAD

**Issue by sliu1000** [CDCgov/prime-reportstream#6445]

**Comment by sliu1000** [comment]
@oslynn  Hi Ott, per your request. I moved this from ticket # 6215 and created a new ticket.

We received an email from (8/8 at 11:35 am) Jake saying that they received 4 more messages. Please review and re-submit.

We have received 4 more messages, showing the following errors. I’m not sure if these belong to ‘Prescryptive’ or not

Filename: covid-19-03a8221b-dea0-4fb0-9fe8-6e732247f6e8-20220808152003

MSH-10: e68c2de4-56a5-484b-b26b-860cc3ffc62b

PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing

PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing

Filename: covid-19-03a8221b-dea0-4fb0-9fe8-6e732247f6e8-20220808152003

MSH-10: 58a4853f-076f-4445-a8db-32edcc438b62

PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing

PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.OrderingFacilityPhoneNumber[0].PhoneNumberString_X Description=Field too long: 18 > 15

PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing

Filename: covid-19-03a8221b-dea0-4fb0-9fe8-6e732247f6e8-20220808152003

MSH-10: 7e7eba8e-7aec-4d63-bde5-b07381f45be6

PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing

PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing

Filename: covid-19-03a8221b-dea0-4fb0-9fe8-6e732247f6e8-20220808152003

MSH-10: 252619e0-37ac-42e2-8bf2-4f5f4bc9e889

PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing

PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing

Please look into this, and re-submit at your earliest convenience. Thanks.
Jake

**Comment by oslynn** [comment]
Slacked SR in prime-reportstream: 2022-08-22:
@Jayna Wallace (CMH/she/her)
 
@sarah
 below are messages that need to be correced:
-----------------------
MSH-10: e68c2de4-56a5-484b-b26b-860cc3ffc62b
PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing (ORC-2.1 need value)
PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing (OBR-2.1 need value)
-----------------------
MSH-10: 58a4853f-076f-4445-a8db-32edcc438b62
PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing (ORC-2.1 need value)
PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.OrderingFacilityPhoneNumber[0].PhoneNumberString_X Description=Field too long: 18 > 15 (ORC-23.1 without extention i.e (217)-555-5555)
PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing (OBR-2.1 need value)
-------------------------
MSH-10: 7e7eba8e-7aec-4d63-bde5-b07381f45be6
PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing (ORC-2.1 need value)
PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing (OBR-2.1 need value)
--------------------------
MSH-10: 252619e0-37ac-42e2-8bf2-4f5f4bc9e889
PATIENT_RESULT[0].ORDER_OBSERVATION[0].ORC.PlacerOrderNumber.EntityIdentifier Description=Required field missing (ORC-2.1 need value)
PATIENT_RESULT[0].ORDER_OBSERVATION[0].OBR.PlacerOrderNumber.EntityIdentifier Description=Required field missing (OBR-2.1 need value)
6:03
Please let me know if it is not clear.

**Comment by oslynn** [comment]
2022-08-23: Emailed senders to resubmit their messages with correct values.

MSH-10: e68c2de4-56a5-484b-b26b-860cc3ffc62b
patricia.mcfain@illinois.gov

MSH-10: 58a4853f-076f-4445-a8db-32edcc438b62	
joni.hull@blessinghealth.org

MSH-10: 7e7eba8e-7aec-4d63-bde5-b07381f45be6	
bherndon@quincymedgroup.com

MSH-10: 252619e0-37ac-42e2-8bf2-4f5f4bc9e889	
stacey@ecofmv.com

**Comment by oslynn** [comment]
2022-08024.  All messages had been updated and resubmitted to MO SFTP site.

Emailed Jake to let him know.
Jake,

I resubmitted following messages with correct values filled ORC-2.1 and OBR-2.1.  I also correct the ORC-23.1 phone format.

MSH-10: e68c2de-56a5-484b-b26b-860cc3ffc62b
MSH-10: 58a4853-076f-4445-a8db-32edcc438b62
MSH-10: 7e7eba8-7aec-4d63-bde5-b07381f45be6
MSH-10: 252619e-37ac-42e2-8bf2-4f5f4bc9e889

Best regards,
-Ott Sathngam
CDC SW ENG.
